src/neo-asm-cli/CommandLineOption.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace neo_asm_cli
+{
+    class CommandLineOption
```

Comment:
Consider using an existing library for command line parsing. NEO Express uses https://github.com/natemcmaster/CommandLineUtils